sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -26,8 +26,7 @@
 from ._xml_workaround_policy import ServiceBusXMLWorkaroundPolicy
 
 from .._common.constants import JWT_TOKEN_SCOPE
-from .._common.utils import parse_conn_str
```

Comment:
can we remove the `parse_conn_str` in `.._common.utils` if it's not used in case it might confuse people in the future "why  there're two parse, which one should I use"

Reply:
Hi @yunhaoling I was thinking about this but didn't know if it was my place to delete code. Will do! Thank you again for the review!

Reply:
We love deleting code when applicable :)  (It's one more line that can't break/doesn't need to be maintained!)  Thanks as well for your patience as we got to this by and by (answering your broader pings inline to this to get two birds with one stone) and to Adam for being 200% on top of all this, he's left me no comments to make! :P

( @yunhaoling I expect we're thinking along the same lines of smoke-testing once the removal is in, just out of paranoia for how core this is? )

Reply:
@KieranBrantnerMagee, that's true, I'm also thinking if removing this would break some tests, but in that case, we shall tweak the test to use the correct `parse_conn_string` to make it happy.

Reply:
Thanks for the follow-up @KieranBrantnerMagee ! FYI @yunhaoling `parse_conn_str` is gone from `.._common.utils` in my latest commit :)